### DISCUSSION TITLE
about instrumentResample

### DISCUSSION THREAD

**Discussion by zdwangjian5** [groops-devs/groops#270] [Q&A]
Dear developers, I want to know how to resample to 0.1s, it seems groops can only interp on multiple seconds not on 0.1 frac seconds?  Firstly, the QT time editor can not set to seconds, even if i  set to 0.1/86400  manually, the groops exe will not return. 
![image](https://github.com/user-attachments/assets/ac376155-aa02-4364-9cbf-d2e8d29e6932)

**Comment by tmayerguerr** [comment]
Hi, 
You are right: the QT time editor is only for convenience and cannot handle every case. You have set the sampling correctly.
I have tested your settings with the program TimeSeriesCreate and the timeSeries class returns the expected result:
```
=== Starting GROOPS ===
Config file: <groops.xml>
--- TimeSeriesCreate (timeSeries.dat) ---
create time series
write instrument data to file <timeSeries.dat>
  time start:      2019-11-01_00-00-00
  time end:        2019-11-03_00-00-00
  epochs:          1728001
  median sampling: 0.1 seconds
  gaps:            0
=== Finished GROOPS ===
```
I therefore cannot see where the problem could be.
regards
Torsten

**Reply by zdwangjian5** [reply]
Thank you for your help! I want to use instrumentResample to interp a timeseries to 0.1s. The instrumentResample function did not return, the log is stoped at 'resample data' as the figure below! Since the data zip is larger than 25MB and it did not support splitted zip files, i can not upload it!  Is there another way to upload the data?Thank you!
![image](https://github.com/user-attachments/assets/8f529049-8eb9-4ba8-9e79-f19623f0bd4b)

**Reply by zdwangjian5** [reply]
Supplyment:  the interp method using 3 order polynomial can return, the function can not return only when using least squares polynomial fit.
